web calculator/web calculator/script.js

```diff
@@ -0,0 +1,3 @@
+function sum(num1 ,num2){
+    return num1+num1
+}
```

Comment:
Nothing ever calls this code so you don't actually do the math. This code just sits unused.